### DISCUSSION TITLE
Notification of accepted OSM edit

### DISCUSSION THREAD

**Discussion by andrewtavis** [organicmaps/organicmaps#2807] [Ideas]
Hello :)

Just came over from Maps.me as the updates there are crazy bad as of late. Firstly, thanks to the team for taking this up (again) and doing it so respectfully and responsibly. Wish I'd realized all the reasons to make the switch earlier. I'm happy to have my old map again :D

Apologies if this idea had been discussed or isn't wanted/possible. Upon finishing my switch to OM, logging in to OSM and looking around a bit, I realized that some of my OSM edits from MM had been accepted. I had a short period where I was adding stuff, but then I never realized that any of them got accepted. This got me thinking that it would have been nice if upon entering the app again I had gotten a simple notification letting me know that something had been accepted. I wouldn't want a push notification to pop up on the app icon, and neither do you all by the looks of it, but just a "5 accepted edits recently, keep it up!" after opening type of thing. I'm thinking a drop down banner from the top that disappears as soon as the user does anything. Not sure given lack of tracking/data collection if there's a way to save and compare total edit counts across sessions, but I was thinking that this might be a way reinforce the behavior of editing OSM through OM - i.e. getting the user to take the extra step of adding a location, rather than just doing a pin if it's not there.

Thanks again so much!

**Comment by biodranik** [comment]
The problem is that OSM accepts all edits automatically :) Well, except those conflicting ones. Or due to some bugs in our code.

Motivating users to edit is a great idea, need to thing how to improve it.

**Reply by biodranik** [reply]
You can see your local edits by typing ?edits in the search field.

Uploaded ones disappear after new maps with those changes are downloaded.

**Comment by andrewtavis** [comment]
Thanks for writing back :) I guess I'm thinking about it from the perspective of a user who doesn't know that the edits will be accepted automatically. I just went through the edit feature, and you are told that it will be checked, then when you go to your profile right afterwards you see that you still have the same amount of edits as before (likely because the system just takes a few minutes to update, as in the time it took to write this I now have the edit counted). 

What this process is missing is "It worked!", because as of now the user is never told that, and checking later does require going into a relatively deep menu location. I always assumed that my change would remain local until accepted. If you had some kind of a banner or other feature to indicate success as soon as their total edit count is increased, then I think that would be a way more welcoming experience to OSM editing.

**Reply by biodranik** [reply]
"It will be checked" is necessary to avoid vandalism. And in reality, it is checked, later, by someone from the community, and can be reverted. By default, all changes should be accepted :)
It's hard to make this feature not too annoying. The idea was "just edit the map and forget about it, it will be uploaded automatically and just work".

**Comment by andrewtavis** [comment]
Makes sense :) Happy to get your feedback on this and learn about the OSM process a bit more! Thanks so much for the conversation 🙏
